MaxText/layers.py

```diff
@@ -441,9 +441,10 @@ def query_init(*args):
     # Layer norms here prevent (near) one-hot softmaxes, which can lead to
     # unstable training loss and nans, see the "QK Normalization" subsection in
     # https://arxiv.org/pdf/2302.05442.pdf.
-    query = LayerNorm(dtype=self.dtype, name='query_layer_norm', kernel_axes = ('heads',))(query)
-    key = LayerNorm(dtype=self.dtype, name='key_layer_norm', kernel_axes = ('heads',))(key)
-    value = LayerNorm(dtype=self.dtype, name='value_layer_norm', kernel_axes = ('heads',))(value)
+    #Llama architecture doesn't have these layernorms
```

Comment:
Probably make these configurable?

Reply:
Yes that seems great but I don't think we want to change types (i.e. None vs. 1e-5). Happy to either have a control flag (qkv_rms_norm_enabled) or use values <0 to mean disabled.